### ISSUE TITLE
SideNav focus ring looks clipped, actually missing left/right border

### ISSUE THREAD

**Issue by lazd** [adobe/spectrum-css#1317]
## Steps to reproduce

1. Go to http://opensource.adobe.com/spectrum-css/
2. Hit tab to the nav items
3. Observe the ring is not complete, it's just on the top and bottom

## Expected behavior

It should be a ring

## Screenshots

![image](https://user-images.githubusercontent.com/201344/141055297-bcbf3b52-1bb2-4a35-ba39-ea0d73c8f1f9.png)
